commands/command_factory.py

```diff
@@ -0,0 +1,7 @@
+from create_command import CreateCommand
+
+class CommandFactory:
+    def create(self, args):
+        cmd = args.command.lower()
+        if (cmd == "create"):
+            CreateCommand(args.schema).execute()
```

Comment:
Factory doesn't execute the command. Factory creates the command only. 

Why its name is `Factory` ? because it is creating.